packages/expansion-hub/src/errors/NoExpansionHubWithAddressError.ts

```diff
@@ -0,0 +1,11 @@
+import {setPrototypeOf} from "./NackError";
+
+export class NoExpansionHubWithAddressError extends Error {
+    moduleAddress: number;
+    constructor(moduleAddress: number) {
```

Comment:
It might be nice to also specify which serial number the parent hub has. Then we can do a message like "Unable to open hub with address X connected via the hub with serial number Y" or something.